spec/database/seed.test.js

```diff
@@ -0,0 +1,124 @@
+const _ = require('lodash');
+const { expect } = require('chai');
+const { organizations } = require('../../database/seed');
```

Comment:
There might be a typo here. I think it should say 'database/seed.js'

Reply:
Thanks! Fixed